app/assets/javascripts/_init-autocomplete.js

```diff
@@ -1,9 +1,18 @@
 jQuery.fn.extend({
   geonameAutocomplete: function () {
     return this.autocomplete({
-      minLength: 2,
-      source: function (request, response) {
-        $.getJSON("/cities/autocomplete?term=" + request.term, function (data) {
+      minLength: 0,
+      // automatically focus first item from autocomplete menu
+      autoFocus: true,
+      source: function(request, response) {
```

Comment:
OK to use old js syntax if you must (a bit odd that heroku is so old school)

Reply:
Asset compilation error, see comments on https://github.com/covoiturage-libre/covoiturage-libre/pull/214